src/Lean/Server/Watchdog.lean

```diff
@@ -342,7 +342,7 @@ def findDefinitions (p : TextDocumentPositionParams) : ServerM <| Array Location
     let srcSearchPath := (← read).srcSearchPath
     if let some module ← searchModuleNameOfFileName path srcSearchPath then
       let references ← (← read).references.get
-      for ident in references.findAt module p.position do
+      for ident in references.findAt module p.position (includeStop := true) do
```

Comment:
This is a behavior change for the `textDocument/definition` request, which had issues analogous to #767. It was a particular issue for renames because the cursor will often be at the end of an identifier span and vscode will give a nonsensical error because it thinks the identifier is being selected and lean doesn't.